### PR TITLE
Enhancement/update structure and other few things

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
This PR adds few things:

1. Fix this warning when launching ayon launcher
    ```
    DEV WARNING: Addon 'colorbleed' does not have defined version.
    ```
2. remove redundant code form server settings
3. update `package.py` and `create_package.py`
4. move `version.py` to the correct location

## Additional Info
`create_package.py` was copied from https://github.com/ynput/example-studio-addon/pull/4

### CODE REVIEW

File: client/ayon_colorbleed/__init__.py [hunk 1]
```diff
@@ -3,9 +3,12 @@
 from ayon_core.addon import AYONAddon
 from ayon_core.addon.interfaces import IPluginPaths
 
+from .version import __version__
 
 class ColorbleedAddon(AYONAddon, IPluginPaths):
     name = "colorbleed"
+    title = "Colorbleed"
+    version = __version__
```
Comment (BigRoy) [inline client/ayon_colorbleed/__init__.py:12]: Am I going crazy - or why does this show as diff? :) The [same line](https://github.com/BigRoy/ayon-colorbleed/blob/main/client/ayon_colorbleed/__init__.py#L11) is already there?

File: package.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name = "colorbleed"
+title = "Colorbleed"
+version = "0.0.4"
```
Comment (BigRoy) [inline package.py]:
```suggestion
version = "0.0.5"
```

File: client/ayon_colorbleed/__init__.py [hunk 3]
```diff
@@ -4,10 +4,17 @@
...
+
+class ColorbleedAddon(AYONAddon, IPluginPaths):
+    name = "colorbleed"
+    title = "Colorbleed"
+=======
 
 
 class ColorbleedAddon(AYONAddon, IPluginPaths):
     name = "colorbleed"
+>>>>>>> main
```
Comment (BigRoy) [inline client/ayon_colorbleed/__init__.py]:
```suggestion


class ColorbleedAddon(AYONAddon, IPluginPaths):
    name = "colorbleed"
    title = "Colorbleed"
```

File: server/__init__.py
Comment (BigRoy) [inline server/__init__.py:1]: Ha - I suppose I did add these in the wrong place somehow? Stupid me. Thanks!

File: client/ayon_colorbleed/__init__.py (hunk 3 above)
Comment (MustafaJafar) [reply client/ayon_colorbleed/__init__.py]: fixed here https://github.com/BigRoy/ayon-colorbleed/pull/1/commits/a2eb135be07db8b9f8d86f921795d3eb424db73d

**Review by BigRoy — APPROVED** [review]
❤️ Thanks!

**Comment by MustafaJafar** [comment]
there's a missing commit although I pushed it 😅

**Comment by BigRoy** [comment]
Whats' going on?

image

EDIT: I suppose Github is lagging

image

**Comment by BigRoy** [comment]
I think we broke Github. Should we just try recreating the PR?

**Comment by MustafaJafar** [comment]
maybe

**Pull request closed without merging**